tensorflow/lite/kernels/split_v_test.cc

```diff
@@ -112,57 +111,93 @@ TEST(SplitVOpTest, TwoDimensional) {
-        {
-            {1, 3, 5, 7, 9, 11, 13, 15},
-            {2, 4, 6, 8, 10, 12, 14, 16},
-        });
+  Check<float, TensorType_FLOAT32>(
+      /*axis=*/0, {2, 2, 2, 2}, {2}, {{1, 2, 2, 2}, {1, 2, 2, 2}},
+      {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, {1, 1},
+      {
+          {1, 2, 3, 4, 5, 6, 7, 8},
+          {9, 10, 11, 12, 13, 14, 15, 16},
+      });
+  Check<float, TensorType_FLOAT32>(
+      /*axis=*/1, {2, 2, 2, 2}, {2}, {{2, 1, 2, 2}, {2, 1, 2, 2}},
+      {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, {1, -1},
+      {
+          {1, 2, 3, 4, 9, 10, 11, 12},
+          {5, 6, 7, 8, 13, 14, 15, 16},
+      });
+  Check<float, TensorType_FLOAT32>(
+      /*axis=*/2, {2, 2, 2, 2}, {2}, {{2, 2, 1, 2}, {2, 2, 1, 2}},
+      {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, {1, 1},
+      {
+          {1, 2, 5, 6, 9, 10, 13, 14},
+          {3, 4, 7, 8, 11, 12, 15, 16},
+      });
+  Check<float, TensorType_FLOAT32>(
+      /*axis=*/3, {2, 2, 2, 2}, {2}, {{2, 2, 2, 1}, {2, 2, 2, 1}},
+      {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, {1, 1},
+      {
+          {1, 3, 5, 7, 9, 11, 13, 15},
+          {2, 4, 6, 8, 10, 12, 14, 16},
+      });
 }
 
 TEST(SplitVOpTest, OneDimensional) {
-  Check(/*axis=*/0, {8}, {8}, {{1}, {1}, {1}, {1}, {1}, {1}, {1}, {1}},
-        {1, 2, 3, 4, 5, 6, 7, 8}, {1, 1, 1, 1, 1, 1, 1, 1},
-        {{1}, {2}, {3}, {4}, {5}, {6}, {7}, {8}});
+  Check<float, TensorType_FLOAT32>(
+      /*axis=*/0, {8}, {8}, {{1}, {1}, {1}, {1}, {1}, {1}, {1}, {1}},
+      {1, 2, 3, 4, 5, 6, 7, 8}, {1, 1, 1, 1, 1, 1, 1, 1},
+      {{1}, {2}, {3}, {4}, {5}, {6}, {7}, {8}});
 }
 
 TEST(SplitVOpTest, OneDimensional2) {
-  Check(/*axis=*/0, {8}, {8}, {{1}, {1}, {1}, {1}, {1}, {1}, {2}, {0}},
-        {1, 2, 3, 4, 5, 6, 7, 8}, {1, 1, 1, 1, 1, 1, 2, -1},
-        {{1}, {2}, {3}, {4}, {5}, {6}, {7, 8}, {}});
+  Check<float, TensorType_FLOAT32>(
+      /*axis=*/0, {8}, {8}, {{1}, {1}, {1}, {1}, {1}, {1}, {2}, {0}},
+      {1, 2, 3, 4, 5, 6, 7, 8}, {1, 1, 1, 1, 1, 1, 2, -1},
+      {{1}, {2}, {3}, {4}, {5}, {6}, {7, 8}, {}});
 }
 
 TEST(SplitVOpTest, NegativeAxis) {
-  Check(/*axis=*/-4, {2, 2, 2, 2}, {2}, {{1, 2, 2, 2}, {1, 2, 2, 2}},
-        {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, {1, 1},
-        {
-            {1, 2, 3, 4, 5, 6, 7, 8},
-            {9, 10, 11, 12, 13, 14, 15, 16},
-        });
+  Check<float, TensorType_FLOAT32>(
+      /*axis=*/-4, {2, 2, 2, 2}, {2}, {{1, 2, 2, 2}, {1, 2, 2, 2}},
+      {1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16}, {1, 1},
+      {
+          {1, 2, 3, 4, 5, 6, 7, 8},
+          {9, 10, 11, 12, 13, 14, 15, 16},
+      });
+}
+
+TEST(SplitVOpTest, TwoDimensionalUint8) {
+  // Input shape: {4, 3}
+  // size_splits: {1, 1, 3}
+  // axis: 0
+  // We should have 3 outpus with shapes respectively:
+  //  output 0 : {1, 3}
+  //  output 1 : {1, 3}
+  //  output 1 : {2, 3}-t
```

Comment:
remove?

Reply:
No removed, just reordered, it is the same, no change.

Reply:
Sorry for confusion. I mean can we remove the '-t' at the line end? What's it for?

Reply:
please also correct the comment here:
output 1: xxx
output 2: xxx
output 3: xxx

same as below.

Reply:
@haozha111: These are not part of my changes, but sure i will handle it, Thanks!